Public/Standalone/Set-PDFForm.ps1

```diff
@@ -1,32 +1,62 @@
 function Set-PDFForm {
-    [CmdletBinding()]
+   [CmdletBinding()]
     param(
-        [string] $SourceFilePath,
-        [string] $DestinationFilePath,
-        [System.Collections.IDictionary]$FieldNameAndValueHashTable
+        [Parameter(Mandatory=$true)]$SourceFilePath,
```

Comment:
You don't need to do Mandatory = $true, by default if it's Mandatory it assumes true. It's cleaner that way